docs/developer_guide/release_process.md

```diff
@@ -72,9 +72,33 @@ For release candidates, check "pre-release".
 
 When the release notes are ready, click `Publish release` on Github. Ludwig's CI will automatically update PyPI.
 
-# 6. Spread the word
+# 6. Update Ludwig docs
 
-Consider sharing the release on LinkedIn/Twitter.
+Check that the [Ludwig PyPi](https://pypi.org/project/ludwig/) has been updated with the newest version.
+
+Go to the ludwig-docs repo and update the auto-generated docs there.
+
+```
+> cd ludwig-docs
+> git pull
+> git checkout -b update_docs
+> pip install ludwig --upgrade
+> python code_doc_autogen.py
+```
+
+If there are any changes, commit them.
+
+```
+> git commit -m "Update auto-generated docs."
+> git push --set-upstream origin update_docs
+```
+
+ Create a PR.
+
+# 7. Spread the word
+
+Announce the release on Slack. If it's a major version release, consider sharing
```

Comment:
Only suggestion I have is the format I've been using on Slack. The idea/premise behind it is that your PR and the updated docs are the long term artifact people can refer to forever while the announcement is just there to draw attention to the change and where to find the complete details.

Here's the format I use:
**What:** Documentation updated for Ludwig {_version_}.
**Why:** _Simple list of updates/changes. For example: Resume parameter for Hyperopt added to documentation._
**Where:** {_Link to updated Ludwig Documentation Page or PR_}

Reply:
Just to clarify -- is your comment regarding the announcement on Slack (step 7) or the description of the PR to update the documentation after release (step 6)?

I don't think it's necessary for the releaser to try to explain specific changes that came from re-generating the MD files via `code_doc_autogen`. Perhaps it's sufficient to link the Ludwig release notes, [e.g.](https://github.com/ludwig-ai/ludwig/releases/tag/v0.5.2). which has the full list of changes in the release.

```
PR title: "Documentation updated for Ludwig vX.Y.Z."
PR description: See [release notes](https://github.com/ludwig-ai/ludwig/releases/tag/v0.5.2) for details.
```

Reply:
Adding a brief description to the actual slack message sounds great to me. I'll add this to the docs.